### PR TITLE
add missing type annotation

### PR DESCRIPTION
**Pull request by tpluscode**
without this tsc complains that `TS7006: Parameter 'err' implicitly has an 'any' type.`

### CODE REVIEW

**Comment by tranvansang** [comment]
@tpluscode 

thanks for the PR.

Did you try installing `@types/express`. The `next`'s type inherits from RequestHandler which is defined in `@types/express`.

I tried compiling the project but could not see the warning you mentioned.

**Comment by tpluscode** [comment]
Yes, I have latest `@types` installed. 

In my project I think it's related to other types which augment the express Request/Response but I couldn't really figure that out.

File: index.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export const combineMiddlewares = (
 	while (Array.isArray(first)) [first, ...middlewares] = [...first, ...middlewares]
 	return (req: Request, res: Response, next: NextFunction) => {
 		if (!first) return next()
-		;(first as RequestHandler)(req, res, err => err
+		;(first as RequestHandler)(req, res, (err: any) => err
```
Comment (tranvansang) [inline index.ts:51]:
```suggestion
		;(first as RequestHandler)(req, res, (err?: any) => err
```

File: index.ts [hunk 2]
```diff
@@ -67,7 +67,7 @@ export const middlewareToPromise = (
 ): Promise<undefined> => new Promise(
 	(resolve, reject) => {
 		try {
-			middleware(req, res, err => {
+			middleware(req, res, (err: any) => {
```
Comment (tranvansang) [inline index.ts:70]:
```suggestion
			middleware(req, res, (err?: any) => {
```

**Comment by tranvansang** [comment]
released in [v1.2.5](https://github.com/tranvansang/middleware-async/releases/tag/v1.2.5)

**Comment by tpluscode** [comment]
thank you

**Pull request merged**